docs/Storage Rules.md

```diff
@@ -0,0 +1,209 @@
+# Storage Rules
+
+File authorization is tricky to manage, and means developers need to spend a lot of time on authentication and authorization. Using Hasura Backend Plus means all this complex code is done for you! All you need to do is set out file access rules, which makes creating and updating rules easy to manage.
+
+The rules are set in a `yaml` file, and let you control granular access to files and folders. Hasura Backend Plus comes with a [template set of rules](https://github.com/nhost/hasura-backend-plus/blob/master/custom/storage-rules/rules.yaml), which you may need to change for your project.
+
+``` yaml
+functions:
+  isAuthenticated: 'return !!request.auth'
+  isOwner: "return !!request.auth && userId === request.auth['user-id']"
+  validToken: 'return request.query.token === resource.Metadata.token'
+paths:
+  /user/:userId/:
+    list: 'isOwner(userId)'
+  /user/:userId/:fileId:
+    read: 'isOwner(userId) || validToken()'
+    write: 'isOwner(userId)'
+```
+
+In the permission variables cookie returned by the [login](../api.md#login) or [refresh](../api.md#refresh-token) endpoints.
```

Comment:
Sentence is not complete.

Reply:
I've moved this down now to be with the functions section. Hopefully it flows better.